tests/gui/components/svg-icon.spec.js

```diff
@@ -72,6 +72,17 @@ describe('Browser: SVGIcon', function () {
       m.chai.expect(element.children().attr('src')).to.equal(imgData)
     })
 
+    it('should use an empty src if there is a parsererror', function () {
+      // The following is invalid, because there's no closing tag for `foreignObject`
+      const iconContents = '<svg><foreignObject></svg>'
+      const imgData = `data:image/svg+xml,`
+      $rootScope.iconContents = iconContents
+
+      const element = $compile('<svg-icon path="iconContents">Resin.io</svg-icon>')($rootScope)
```

Comment:
Apologies for being super-dense (probably due to me having zero knowledge of Angular!), but I'm still struggling to understand why this line doesn't need string-interpolation, but lines 88, 96, etc. do :confused: